src/components/HfpLayer.js

```diff
@@ -2,16 +2,27 @@ import React, {Component} from "react";
 import {Polyline, Popup, CircleMarker, Tooltip} from "react-leaflet";
 import {latLng} from "leaflet";
 import get from "lodash/get";
+import groupBy from "lodash/groupBy";
+import orderBy from "lodash/orderBy";
+import map from "lodash/map";
+import random from "lodash/random";
 import moment from "moment";
 
+const identities = {};
+
 class HfpLayer extends Component {
   mouseOver = false;
 
-  coords = this.props.positions
-    .filter((pos) => !!pos.lat && !!pos.long)
-    .map(({lat, long, ...rest}) => {
-      return [lat, long, rest];
-    });
+  coords = orderBy(
+    this.props.positions.filter((pos) => !!pos.lat && !!pos.long),
+    (pos) => moment(pos.receivedAt).unix()
+  ).map(({lat, long, ...rest}) => {
+    return [lat, long, rest];
+  });
```

Comment:
This component is temporarily not in use, so I haven't updated the coords processing. They are ordered in Root, so this should definitely be updated.